pylint/checkers/design_analysis.py

```diff
@@ -81,11 +82,9 @@
             "Used when an if statement contains too many boolean expressions.",
         ),
         "R0917": (
-            "Too many positional arguments in a function call.",
-            "too-many-positional",
-            "Will be implemented in https://github.com/pylint-dev/pylint/issues/9099,"
-            "msgid/symbol pair reserved for compatibility with ruff, "
-            "see https://github.com/astral-sh/ruff/issues/8946.",
+            "Too many positional arguments (%s/%s)",
+            "too-many-positional-arguments",
```

Comment:
I also prefer the new name. But Ruff chose the other one and we originally created this message in anticipation for compatibility with ruff. Is it worth the possible confusion ?

Reply:
We can easily rename. `ruff` only supports codes for now, so renaming is just a change in output and documentation. That should be fine for them? Considering https://github.com/astral-sh/ruff/issues/8946 was merged by @charliermarsh perhaps they can confirm that a rename should be fine for them as well?

Reply:
They fixed it in 0.6.1 / https://github.com/astral-sh/ruff/pull/12905. There's a comment saying we should have named it ``"too-many-positional-parameters`` (https://github.com/astral-sh/ruff/issues/12619#issuecomment-2265121518).

Reply:
I get it. But leaving it this way maintains the rhyme between the existing `max-args` setting and the new `max-positional-arguments` setting.

Reply:
Yeah and the churn in ruff and pylint if we change it again would be pretty bad. (tbh I didn't consciously make this distinction between arguments/parameters internally myself)